### ISSUE TITLE
[SECURITY] [BUG] Why is latitude + longitude even in the codebase? There are many ways to accomplish the same UX, no SSN or lat lon even mentioned in the codebase. Bet.

### ISSUE THREAD

**Issue by sam1rm-hiretech** [darkreader/darkreader#4348]
### **Describe the bug**

Using this extension on Edge, and inspecting the codebase, the latitude and longitude are way too easily accessible. Too easy + it's not necessary if you are using the Locale TZ javascript libraries to figure out the time zone to provide a better experience. 
### **Steps to reproduce**

Use this extension. Go to inspect element or browser developer tools. CTRL + F => Latitude. Oops. 

### **Expected behavior**

Latitude and longitude should not be in the codebase period. Sorry. 

### **Actual behavior**

Figure out local time zones without latitude and longitude. Have the user upload their location or share it with you explicitly to retain that data. Cannot be the default. That is ridiculous?. 

### **Screenshots**

Really? Ok. I'll do this when I get home. @sam1rm

### **System Information:**

- OS: Windows 10
- Browser: Latest edge.
- Dark Reader version: I installed and uninstalled this today. idk. 

### **Additional context**

Digging deeper, it's clear there are good intentions from the very beginning. https://github.com/darkreader/darkreader/issues/72

Curious how this will be received in 2020 - a lot has changed since then.

**Comment by Gusted** [comment]
Hi @sam1rm-hiretech!

It seems like you got a lot of concerns.
- You used the Bug format so that's why the Screenshot is a section, please review our [security policy](https://github.com/darkreader/darkreader/security/policy) next time if you want to report an security.
- As I'm reading this, their is an concern that when using the Latitude + Longitude to receive up-to-date dark/day that the Latitude + Longitude is available. However we uses the `web Storage API` whereby the settings are only available to Dark Reader extension scope. That limits how the settings can be retrieved. However the settings are still saved on a remote server(If the users has an setting sync on the current browser) or locally.
- I just want to point out that the Latitude + Longitude option is totally optional and not required at all and users should have consent if they use this information that it can be possible used and I think a consent warning should be in place for this actually, not sure.
- I just wanted to say that your attitude on this issue didn't really sparked me at all.
- `Curious how this will be received in 2020 - a lot has changed since then.` It still *totally* optional and the information is not collected and a few years back it seems to be a big feature request, so my best guess those people are enjoying it.
- `Latitude and longitude should not be in the codebase period. Sorry.` I disagree, no sorry.
- `Figure out local time zones without latitude and longitude ` That could be a good suggestion to use local time zones however time zones don't give a good indication when it's dark as it's only spliced vertical it could be Finland or South-Africa, same time-zone but totally totally different sun rises and sun dawns. 
- `Have the user upload their location or share it with you explicitly to retain that data.` Please refer to `The user has saved it's location into the extension's settings` As it's not shared **with me as an person** and the extension's developer cannot access those data, It's either on the local machine or on the sync server if applicable.
- `Cannot be the default. That is ridiculous?.` Default? Let me clear this once again up. **It's an >>optional<< option to choose for if you like to change Dark Reader's theme when the sun sets and sun rises**, Users likes to have this option available as you can read in #72. 

You wanted an exhausted answer? Please read it carefully and hopefully clears up some misunderstanding.

**TL;DR**
*I'ts an optional option that users likes to have, however I can agree that a consent/warning that the data will be saved locally or on a remote server non-encrypted is a good idea.* 

CC @erkinalp as security policy writer.

Regards,
Gusted

**Comment by Gusted** [comment]
> no SSN or lat lon even mentioned in the codebase. Bet.

https://github.com/darkreader/darkreader/search?q=latitude Did you maybe had a typo while writing it? I see results in the code base for the word `latitude`.

Regards,
Gusted

**Comment by Gusted** [comment]
No response.
